salt/ext/importlib_metadata/__init__.py

```diff
@@ -41,15 +41,22 @@
     NullFinder,
     PyPy_repr,
     install,
-    Protocol,
+    # Protocol,
 )
 
 from configparser import ConfigParser
 from contextlib import suppress
 from importlib import import_module
 from importlib.abc import MetaPathFinder
 from itertools import starmap
-from typing import Any, List, Optional, TypeVar, Union
+#from typing import Any, List, Optional, TypeVar, Union
```

Comment:
removed?

Reply:
Same as above, and yes most of the code was left as comment